### PR TITLE
Bump tektoncd/pipeline to v0.39.0

### PR DESCRIPTION
**Pull request by piyush-garg**
This will bump tektoncd/pipeline to v0.39.0
and do the necessary changes in code

# Submitter Checklist

These are the criteria that every PR should meet, please check them off as you
review them:

- [ ] Includes [tests](https://github.com/tektoncd/community/blob/master/standards.md#principles) (if functionality changed/added)
- [ ] Run the code checkers with `make check`
- [ ] Regenerate the manpages, docs and go formatting with `make generated`
- [ ] Commit messages follow [commit message best practices](https://github.com/tektoncd/community/blob/master/standards.md#commit-messages)

_See [the contribution guide](https://github.com/tektoncd/cli/blob/master/CONTRIBUTING.md)
for more details._

# Release Notes

```release-note
Bump tektoncd/pipeline to v0.39.0
```

### CODE REVIEW

**Review by pradeepitm12 — APPROVED** [review]

File: pkg/pods/pod_template.go [hunk 1]
```diff
@@ -18,12 +18,12 @@ import (
...
 	"github.com/tektoncd/cli/pkg/file"
-	"github.com/tektoncd/pipeline/pkg/apis/pipeline/v1beta1"
+	"github.com/tektoncd/pipeline/pkg/apis/pipeline/pod"
 	"sigs.k8s.io/yaml"
 )
 
-func ParsePodTemplate(httpClient http.Client, podTemplateLocation string, validate file.TypeValidator, errorMsg error) (v1beta1.PodTemplate, error) {
-	podTemplate := v1beta1.PodTemplate{}
+func ParsePodTemplate(httpClient http.Client, podTemplateLocation string, validate file.TypeValidator, errorMsg error) (pod.PodTemplate, error) {
+	podTemplate := pod.PodTemplate{}
```
Comment (vdemeester) [inline pkg/pods/pod_template.go:26]: Arf.. it should have been `pod.Template`.. but oh well.

**Review by vdemeester — APPROVED** [review]
/lgtm

File: pkg/pods/pod_template.go (hunk 1 above)
Comment (vdemeester) [reply pkg/pods/pod_template.go:26]: Oh actually that's the case already, `PodTemplate` is an alias to `Template`. Ideally we would use `Template` there.

Comment (piyush-garg) [reply pkg/pods/pod_template.go:26]: they have both :D

**Pull request merged**